### PR TITLE
updated for final pres

### PR DESCRIPTION
**Pull request by snoozan**

### CODE REVIEW

File: scripts/people/2014/fall/susan.yaml [hunk 1]
```diff
@@ -26,6 +26,7 @@ hw:
   hackathon2: http://emollient.me/election-day-hackathon.html
   hackathon3: http://emollient.me/girl-develop-it.html
   hackathon4: http://emollient.me/vimeo-presentation.html
+  finalpreso: http://www.csh.rit.edu/~slunn/geomunchers-pres/presentation.html#/
```
Comment (ryansb) [inline scripts/people/2014/fall/susan.yaml]: FYI: the `#` is a comment in YAML, so that `#/` on the end won't get rendered in the link.

**Comment by snoozan** [comment]
Fixed.

File: scripts/people/2014/fall/susan.yaml [hunk 2]
```diff
@@ -5,6 +5,15 @@ forges:
...
 rit_dce: skl1958
+
+hackathon0: http://emollient.me/software-freedom-day-2014.html
+hackathon1: http://emollient.me/big-red-hacks-at-cornell.html
+hackathon2: http://emollient.me/election-day-hackathon.html
+hackathon3: http://emollient.me/girl-develop-it.html
+hackathon4: http://emollient.me/vimeo-presentation.html
+hackathon5: http://emollient.me/rochester-mini-maker-faire.html #attended as a volunteer
+hackathon6: http://emollient.me/homestretch-hackathon.html
+playtesting2: http://emollient.me/playtesting-2.html
```
Comment (decause) [inline scripts/people/2014/fall/susan.yaml]: s/playtesting/playtest

**Comment by decause** [comment]
:lock_with_ink_pen:

**Pull request merged**